### DISCUSSION TITLE
How to list documents that I have update permission

### DISCUSSION THREAD

**Discussion by nexmoe** [appwrite/appwrite#4579] [Q&A]
Now, I can list all documents that I have read permission.
But, sometimes I don't have update permission or delete permission.
I want to filter out the documents that I have update permission or delete permission.

**Comment by stnguyen90** [comment]
Unfortunately, you can't query on permissions at the moment. The typical pattern for this, however, is to query on another attribute. For example, let's say there's a blog posts collection and the person who created the blog post should be able to update and delete their post. I would add a `createdBy` attribute to the blog post to store the creator and then query on that attribute.

**Reply by nexmoe** [reply]
But in this case, the user can define his `createdBy` attribute arbitrarily.
